### DISCUSSION TITLE
Is it possible to disable hollow cursor when window is unfocused?

### DISCUSSION THREAD

**Discussion by SolsticeSpectrum** [kovidgoyal/kitty#5689] [Q&A]
![image](https://user-images.githubusercontent.com/38611351/202779698-22d4f90d-f509-442a-a2ee-bcd66db66d59.png)

The caret is already a good indicator for me to know which terminal is selected. I would rather like the hollow cursor to just be invisible.

**Comment by kovidgoyal** [comment]
No, it isnt

**Reply by SolsticeSpectrum** [reply]
Could it be added as config? I tried it myself but I have no idea where **which** in **render_cursor** is being defined to 1, 2, 3

**Comment by kovidgoyal** [comment]
Not something I am interested in, sorry.
